docs/userguides/dependencies.md

```diff
@@ -1,6 +1,6 @@
 # Dependencies
 
-Ape downloads and caches dependencies in the `.ape/packages/<name>/<version-id>` directory where `<name>` refers to the name of the dependency and `<version-id>` refers to the version or branch of the package.
+Ape downloads and caches dependencies in the `.ape/packages/projects/<name>/<version-id>` directory where `<name>` refers to the name of the dependency and `<version-id>` refers to the version or branch of the package.
```

Comment:
What other folders appear under there now?

Reply:
`apis`: for caching dependency APIs, which is needed for using dependencies outside of a project, being able to fetch a dependency outside a project, makes the CLI commands just work better in all regard... 

`manifests`: which are the individual manifest for each dependency project, works the same `__local__.json` for the local one.

Reply:
I was thinking that we can use "global names" (the names that each dependency type uses e.g. `github: OpenZeppelin/openzeppelin-contracts` instead of the "local names" (the name we give to the dependency within the project) inside this directory. Those "local names" are really only important when referencing them inside of source remappings for the project using the compiler plugin config

Reply:
I think that makes sense but it might be more clear then to have "name: " in the config be "import_key: " or "remapping_key: " or something, rather than have 2 names

Reply:
Done! Storage keys of dependencies are way better now.